openapi/multi-yaml/components/schemas/CheckoutCompletion.yaml

```diff
@@ -7,19 +7,19 @@ properties:
   payment_mode:
     type: string
     example: ecom
-    enum: [ecom]
+    enum: [ecom, bnpl, card_present]
   payment_token:
     type: string
     example: pm_xyz123
-    description: the payment token used to process the payment, only for ecom payments
+    description: the payment method token used to process the payment, only for ecom payments
   status:
     type: string
     enum: [succeeded, failed]
     example: succeeded
     description: the status of the completion, only succeeded or failed
   payment_status:
     type: string
-    enum: [succeeded, failed, pending, canceled]
+    enum: [succeeded, failed, pending, canceled, skipped]
```

Comment:
The checkout completion model lists `skipped` as one possible `payment_status` so I added it here.